R/tt_compatibility.R

```diff
@@ -407,6 +433,11 @@ rbindl_rtables <- function(x, gap = 0, check_headers = TRUE) {
 #' @param deparse.level numeric(1). Currently Ignored.
 #' @param \dots ANY. Elements to be stacked.
 #'
+#' @note 
+#' When objects are rbinded, titles and footer information are retained from the first object (if any exist)
```

Comment:
Looks like this text is a mix of two different versions, please fix to accurately describe the behavior